examples/benchmark_xgboost.py

```diff
@@ -0,0 +1,1100 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
```

Comment:
@haozhenWu Think the PR will be good to go once this script and `xgboost_results.csv` are removed. Thanks again for your hard work, good to have new models and contributors!